docs/concepts/pyarrow.md

```diff
@@ -53,12 +53,12 @@ First we define two source streams, made of `pyarrow.Table`:
 --8<-- "examples/pyarrow_concepts.py:dag_source"
 ```
 
-Then we keep track of the latest value for each source stream:
+Then we keep track of the latest value for each source streams:
```

Comment:
nitpick: each ... stream should be singlar, no?

Reply:
indeed.